### DISCUSSION TITLE
PCM 5 Porsche Macan 2020

### DISCUSSION THREAD

**Discussion by AKALuffy** [Mr-MIBonk/M.I.B._More-Incredible-Bash#402] [General]
Hello Everyone,

Thanks for your amazing work.

Just a few question : 

I that compatible for porsche macan 2020 MH2p_ER_PO416_P2656 ?

I saw kind of image here : https://mibsolution.one/#/1/9/MH2P%20-%20ALPINE/PORSCHE/MACAN

**Comment by yaspardo** [comment]
Hey @AKALuffy 

Is it? Mine is Macan S 2020 MH2p_ER_PO416_K2874 and wondering if it is compatible!

**Comment by AKALuffy** [comment]
Nope its not

**Comment by yaspardo** [comment]
Do you mind if I ask whether there are any plans to release a patch for PCM 5 that enables full-screen Apple CarPlay as well?

**Reply by mcflexus** [reply]
There already is and there is a different toolbox for the pcm 5, search for mh2p toolbox it does both enable aa/ca and fullscreen.

**Comment by yaspardo** [comment]
Thank Felix for your response. 

My MIB is MH2p_ER_PO416_K2874. (UK based owner of 2nd Gen Macan S 2020YM) 
Is the one shown in the screenshot below the correct one?


￼

Thanks,
Yashar

**Reply by mcflexus** [reply]
Both should be good. If you can't find it let me know, i can send files then.

**Comment by yaspardo** [comment]
Thanks a lot Felix,

I’ve already downloaded 

MH2p-ER-PO416_P9830 from

https://mibsolution.one/index.php/f/165806

with your confirmation, I suppose this is the right one.

Already ordered SD card reader to prepare the SD card. 

I am unsure how to start the MIB installation once I insert the SD card into the unit. I have checked all relevant guides on mibwiki.one to avoid asking unnecessary  questions, but I couldn’t find specific hidden menu key combinations and shortcuts for PCM5.0. 


https://mibwiki.one/doc/key-combinations-and-shortcuts-7tk8NfVoLo￼

So to summarize: 

1- MH2p-ER-PO416_P9830.rar downloaded 
2- SD card will be formatted with FAT32
3- Content will be unpacked onto the SD card root
4- SD Card will be inserted into the MIB unit
5- How installation process can be triggered?

Hope I got it right.

Regards,
Yashar

**Reply by mcflexus** [reply]
You're quite wrong on this one. You downloaded an update for the pcm. The mh2p toolbox is run from a laptop connected to your car with a specific ethernet adapter connected to your car usb. You have to put the files from dowload link on sd. Everything is explained in the readme [https://disk.yandex.com/d/7WKZix6ocBNLCw](url)

**Comment by yaspardo** [comment]
Thanks Felix. The link is bad. Could you please send it again?

I have already downloaded the tool in the link below

https://mibwiki.one/doc/toolbox-1XMSZakmGT

Are these the steps I need to follow for full-screen Apple CarPlay?

Thanks,
Yashar

**Comment by mcflexus** [comment]
Yes that's the one, make sure to read the readme inside the zip it's more recent and a little different.

**Comment by yaspardo** [comment]
Thanks Felix, very helpful indeed.

I thoroughly studied the README of MH2p_Toolbox_v1.4.3_by_Enthusiast as well as https://www.drive2.ru/b/644918546345774500/ a few times and am wondering if updating the train from MH2p_ER_PO416_K2874 to MH2p_ER_PO416_P2865 is compulsory for enabling fullscreen CarPlay. 
As per my Porsche service center, K2874 is the latest FW version. 

Only intending to perform function 5: Enable fullscreen CarPlay.
Is it supported with my existing train (MH2p_ER_PO416_K2874)? 
Is there any caveat to be aware of? & finally is it reversible?

I suppose in the SD card I need to transfer the Toolbox folder (MH2p_Toolbox_v1.4.3_by_Enthusiast) content to the root so toolbox.sh can be run from the root. 

Regards,
Yashar

**Comment by asunnyhero** [comment]
@yaspardo did you manage to figure out whether the firmware update was possible?

**Comment by yaspardo** [comment]
@asunnyhero Haven't tried and still wondering what are the benefits of updating to P2865 and if it is complusary for only enabling fullscreen CarPlay.

**Reply by asunnyhero** [reply]
Thanks, if all you're looking for is FS CarPlay then this should do the trick.  I've used it in a 2019 Macan S and 2019 992 CS, and have it working for CarPlay and Android Auto in both windowed and fullscreen modes.

https://rennlist.com/forums/cayenne-9y0-2019/1440606-free-easy-pcm-5-full-screen-carplay.html

**Reply by asunnyhero** [reply]
Meant to say, this doesn't need a FW update to work

**Reply by yaspardo** [reply]
Thanks a lot. It worked smoothly

**Comment by yaspardo** [comment]
Just managed to enable full screen Apple CarPlay using the instructions provided. Thanks a lot for the heads up mate. Regards,Yashar On 27 Nov 2025, at 17:26, asunnyhero ***@***.***> wrote:﻿
Meant to say, this doesn't need a FW update to work
